package/CHANGELOG

```diff
@@ -22,6 +22,8 @@ mm/dd/yy richardjgowers, kain88-de, lilyminium, p-j-smith, bdice, joaomcteixeira
   * 0.21.0
 
 Fixes
+  * Checks for cryo-em 1 A^3 default CRYST1 record, 
```

Comment:
You also need to add yourself to the list above (line 20).

Reply:
Done. 😅 I guess I'm having issues rebasing right now. Could you take care of the merge conflicts? Thanks.

Reply:
Sure now worries 👍

Reply:
Should I close this?

Reply:
*this PR

Reply:
No need to do anything, I'll be merging the PR once Travis CI completes. (I'm just checking that everything is all done & we didn't break anything since it last ran).